readthedocs/core/utils/general.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+import os
+
+from django.shortcuts import get_object_or_404
+
+from readthedocs.core.utils import broadcast
+from readthedocs.projects.tasks import remove_dirs
+from readthedocs.builds.models import Version
+
+
+def wipe_version_via_slug(version_slug):
+    """Wipes the given version."""
+    version = get_object_or_404(Version, slug=version_slug)
```

Comment:
This needs to be filtering by a project, not just a version slug.

Reply:
Okay.
I have pushed the changes.